### PR TITLE
fix: update react hook to return undefined and appBridge

### PR DESCRIPTION
**Pull request by julianiff**
…ws easier handling in the app. 

@getflourish 

Now we can do 
```
const appBridge = usePlatformAppBridge();
[…]
const context = appBridge?.context().get();
```

### CODE REVIEW

File: packages/app-bridge/src/react/usePlatformAppBridge.spec.ts [hunk 1]
```diff
@@ -19,18 +19,14 @@ describe('usePlatformAppBridge', () => {
...
-    it('should return initiated false', async () => {
-        const { result } = renderHook(() => usePlatformAppBridge());
-        expect(result.current.connected).toBe(false);
-    });
-    it('should return initiated true after waiting', async () => {
+    it('should return platformApp after initiation and waiting', async () => {
         const { result } = renderHook(() => usePlatformAppBridge());
         await waitFor(() => {
-            expect(result.current.connected).toBe(true);
+            expect(result.current).toBeTypeOf('object');
```
Comment (marguslant0) [inline packages/app-bridge/src/react/usePlatformAppBridge.spec.ts:29]: Would it make sense to also add the earlier
```
expect(result.current.platformAppBridge).toBeDefined();
```
to make it more specific?

Comment (julianiff) [reply packages/app-bridge/src/react/usePlatformAppBridge.spec.ts:29]:
```suggestion
            expect(result.current).toBeDefined();
```

**Review by mike85 — APPROVED** [review]
LGTM!

**Review by marguslant0 — APPROVED** [review]
image

**Pull request merged**